<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawDlg</class>
 <widget class="QDialog" name="DrawDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>681</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>绘图</string>
   </property>
   <widget class="DrawWidget" name="drawWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>661</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comBoxStyle">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>线条样式</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPenWidth">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>1</y>
       <width>46</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>线宽</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comBoxShape">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonColor">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUndo">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>撤回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>5</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>形状</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header location="global">drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
